<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>571</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Задание 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Решение</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>a</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>b</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>c</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Задание  2</string>
     </attribute>
     <widget class="QRadioButton" name="radSelButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Радианы</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="angel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Угол</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Сторона A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Сторона B</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="side_A">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="side_B">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="resulte_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Ex_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Решение</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Задание 3</string>
     </attribute>
     <widget class="QTextEdit" name="Text_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Text_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Replace_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Кнопка 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Random">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Кнопка 3</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="Move_Text">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
